crates/matrix-sdk-common/src/serde_helpers.rs

```diff
@@ -80,9 +82,9 @@ pub fn extract_bundled_thread_summary(
     match event.get_field::<Unsigned>("unsigned") {
         Ok(Some(Unsigned { relations: Some(Relations { thread: Some(bundled_thread) }) })) => {
             // Take the count from the bundled thread summary, if available. If it can't be
-            // converted to a `u64`, we use `UInt::MAX` as a fallback, as this is unlikely
+            // converted to a `u64`, we use `usize::MAX` as a fallback, as this is unlikely
```

Comment:
This doesn't make sense. `UInt::MAX` is always representable as `u64`. Looks like this comment has been wrong about what happens here for a while.


```suggestion
            // converted to a `usize`, we use `usize::MAX` as a fallback, as this is unlikely
```

Reply:
Well, this is an _addition_ to the edit made by @zzorba. This is fixing a pre-existing issue with the comment that @zzorba missed, does not undo any of his changes.